addons/mrp/models/mrp_production.py

```diff
@@ -382,7 +381,14 @@ def _compute_components_availability(self):
         # Force to prefetch more than 1000 by 1000
         all_raw_moves._fields['forecast_availability'].compute_value(all_raw_moves)
         for production in productions:
-            if any(float_compare(move.forecast_availability, 0 if move.state == 'draft' else move.product_qty, precision_rounding=move.product_id.uom_id.rounding) == -1 for move in production.move_raw_ids):
+            if any(
+                float_compare(
+                    move.forecast_availability,
+                    0 if move.state == 'draft' else move.product_qty,
+                    precision_digits=self.env['decimal.precision'].precision_get('Product Unit of Measure')
```

Comment:
Move the `precision_get` outside of the loop, especially since it's meant to be run by a *lot* of records :slightly_smiling_face: